lib/text_helpers/translation.rb

```diff
@@ -13,6 +13,7 @@ def link(link, title, content)
         ("href=\"#{link}\"" if link),
         ("title=\"#{title}\"" if title),
         ("target=\"_blank\"" if link =~ PROTOCOL_MATCHER),
+        ("rel=\"noopener\"" if link =~ PROTOCOL_MATCHER),
```

Comment:
Can we pull this check outside of the attribute composition? Regex matching is fast, but there's no need to run the same check twice in this method. Additionally, since we're only explicitly supporting Ruby 2.6+ at this point, it probably makes sense to use `link.match?(PROTOCOL_MATCHER)` now, which was added way back in 2.4.

Reply:
Updated and force-pushed.